src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -496,6 +497,29 @@ def contains_complete_query(span: Span, is_source: Optional[bool] = False) -> bo
         return query and not query.endswith("...")
 
 
+def total_span_time(span_list: List[Dict[str, Any]]) -> float:
```

Comment:
I just burned myself pretty good with assuming that a `float` that a function was returning was in milliseconds, but it was seconds, so I'm scared of this `-> float`. I think a `timedelta` is better, but that's not a strong opinion!